Server/controllers/index.js

```diff
@@ -1,11 +1,23 @@
 const router = require('express').Router();
-const {  getLastBill, getUserBillType } = require('./Bill');
+const {
+  getLastBill,
+  getUserBillType,
+  newBill,
+  getBillType,
+} = require('./Bill');
 const { logIn } = require('./User');
 const { getPersonalInfo } = require('./User');
 
-router.get('/api/bill/:bill_type', getLastBill )
-router.post('/api/login', logIn);
-router.get('/api/profile', getPersonalInfo);
-router.get('/api/home', getUserBillType);
+const { getProviderByType } = require('./Provider');
+
+// router.get('/api/bill/getBillByType/:bill_type', getLastBill);
+// router.post('/api/login', logIn);
+// router.get('/api/profile', getPersonalInfo);
+// router.get('/api/home', getUserBillType);
+
+// router.get('/api/getBillType', getBillType);
```

Comment:
why do comments for this endpoints

Reply:
sorry , comments for tested and forgot to return it